_quickstarts/deploy.markdown

```diff
@@ -0,0 +1,119 @@
+
+In GitHub, all releases since v0.4.0 have an attached `kroxylicious-app-*-bin.zip` file. Download the latest version of this zip, and optionally verify the contents of the package with the attached `kroxylicious-app-*-bin.zip.asc` file.
+
+{% capture os_archive_note %}
+If you're trying Kroxylicious out on Linux or macOS, you may find the `.tar.gz` format easier to work with. We're using the `.zip` files in this quickstart for cross-platform compatibility, but we recommend you use whichever format you're most familiar with.
+{% endcapture %}
+
+{% include bs-alert.html type="primary" icon="info-circle-fill" content=os_archive_note %}
+
+<br />
+
+# Install
+
+Extract the downloaded Kroxylicious Zip file into the directory you would like to install Kroxylicious in.
+Ensure the `kroxylicious-start.sh` and `run-java.sh` files in the `bin/` directory within the extracted folder have at least read and execute (`r-x`) permissions for the owner.
+
+<br />
+
+# Configure
+
+Kroxylicious is configured with YAML. An example configuration file can be found in the `config/` directory of the extracted Kroxylicious folder, which you can either modify or use as reference for creating your own configuration file.
+
+From the configuration file you can specify how Kroxylicious presents each Apache Kafka&#174; broker to clients, where Kroxylicious will locate the Apache Kafka&#174; cluster(s) to be proxied, and which filters Kroxylicious should use along with any configuration for those filters.
+
+For this quickstart we will use Kroxylicious in Port-Per-Broker configuration, and assume that both your Apache Kafka&#174; cluster and clients are running on your local machine and using their default configuration. This means we can use the example proxy config file that comes with Kroxylicious.
+
+{% capture kroxylicious_config_note %}
+If your machine uses a non-standard port configuration, or if you have used custom settings for your Apache Kafka&#174; cluster (or if your cluster is running on a different machine) you will need to adjust your Kroxylicious configuration accordingly. More information about configuring Kroxylicious can be found in the [documentation](https://kroxylicious.io/kroxylicious/#_deploying_proxies).
+{% endcapture %}
+
+{% include bs-alert.html type="primary" icon="info-circle-fill" content=kroxylicious_config_note %}
+
+<br />
+
+# Run
+
+From within the extracted Kroxylicious folder, run the following command:
+
+```shell
+./bin/kroxylicious-start.sh --config config/example-proxy-config.yml
+```
+
+To use your own configuration file instead of the example, just replace the file path after `--config`.
+
+<br />
+
+# Use
+
+To use your Kroxylicious proxy, your client(s) will need to point to the proxy (using the configured address) rather than directly at the Apache Kafka&#174; cluster.
+
+Here's how you would use Kroxylicious with the console producer and console consumer:
```

Comment:
```suggestion
Here's how you would use Kroxylicious with the command line clients from the Apache Kafka&#174; distribution:
```

It sounds a bit clumsy, but it hints at where to get the tools from and covers the fact the first script used isn't actually the console producer or consumer.

Reply:
We're already pointing users to the Apache Kafka quickstart at the start of this doc, which involves using these clients. If we still feel the need to point this out here specifically, I might instead use the phrasing
```suggestion
Here's how you would use Kroxylicious with the Apache Kafka&#174; command line clients:
```

Reply:
I think it needs a little more context here, after all people don't read, they skim at best. Your version certainly reads better than mine.